framework/tests/all-systems/Enums.java

```diff
@@ -13,18 +14,18 @@ void foo2(MyEnumSet<ElementType> p) {}
 
   <E extends Enum<E>> void mtv(Class<E> p) {}
 
-  <T extends Object> T checkNotNull(T ref) { return ref; }
```

Comment:
I fixed a bunch of all-systems tests and ignored many errors in them. I think the core behavior of the Lock Checker is captured in the Lock Checker tests. For these scenarios involving more complex types, we already know our support in the Lock Checker is hacky.